lldb/include/lldb/API/SBProcess.h

```diff
@@ -209,6 +209,14 @@ class LLDB_API SBProcess {
 
   lldb::addr_t ReadPointerFromMemory(addr_t addr, lldb::SBError &error);
 
+  lldb::SBAddressRangeList
+  FindRangesInMemory(const void *buf, uint64_t size, SBAddressRangeList &ranges,
+                     uint32_t alignment, uint32_t max_matches, SBError &error);
+
+  lldb::addr_t FindInMemory(const void *buf, uint64_t size,
+                            SBAddressRange &range, uint32_t alignment,
```

Comment:
Could you make `range` const?

Reply:
We have gone back and forth between making it const/non-const. I have updated it. Tagging @clayborg for awareness.